mapie/tests/test_classification.py

```diff
@@ -1744,12 +1729,9 @@ def test_regularize_conf_scores_shape(k_lambda) -> None:
     Test that the conformity scores have the correct shape.
     """
     lambda_, k = k_lambda[0], k_lambda[1]
-    args_init, _ = STRATEGIES["raps"]
-    clf = LogisticRegression().fit(X, y)
-    mapie_clf = MapieClassifier(estimator=clf, **args_init)
     conf_scores = np.random.rand(100, 1)
     cutoff = np.cumsum(np.ones(conf_scores.shape)) - 1
-    reg_conf_scores = mapie_clf._regularize_conformity_score(
+    reg_conf_scores = RAPS._regularize_conformity_score(
```

Comment:
this now test the conformity score and not the MapieClassifier class, so this test should go in another file

Reply:
Done 👍